inc/requesttype.class.php

```diff
@@ -39,7 +39,7 @@ class RequestType extends CommonDropdown {
 
 
    static function getTypeName($nb = 0) {
-      return _n('Request source', 'Request sources', $nb);
+      return _nx('requesttype', 'Request source', 'Request sources', $nb);
```

Comment:
Not sure a context is usefull.
There are 2 contexts :
1. itil object request source;
2. an obscure usage of `__('Request source')` in `Item_Software*` that looks more as a bad copy/paste than an intended usage of this string.

Reply:
I've removed context here, added on the other side (since I'm currently not sure what the correct string would be), and replaced all hardcoded translation wth getTypeName call.